index.js

```diff
@@ -1,3 +1,7 @@
 function subtract(a, b) {
   return a - b;
+}
+
+function division(a, b) {
+  return a / b;
```

Comment:
Olá @maite-marques! Tudo bom? :smile:

Talvez seja bom prever os casos em que a o `b` seja igual a zero, o que você acha?  

```suggestion
  return a / b;
```